src/constants/auth/index.js

```diff
@@ -0,0 +1,11 @@
+export const CONFIRM_PASSOWRD_ERROR = 'Passwords do not match.';
```

Comment:
Follow this naming convention

export const VALIDATION_CONFIRM_PASSWORD
export const VALIDATION_FIRST_NAME

export const SUCCESS_SIGNUP

export const VALIDATION_USERNAME